src/GrasshopperTeklaDrawingLink/Components/Text/Arrow component.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Grasshopper.Kernel;
+
+using GTDrawingLink.Properties;
+using GTDrawingLink.Tools;
+using GTDrawingLink.Types;
+
+using System.Collections.Generic;
+using System.Drawing;
+
+using Tekla.Structures.Drawing;
+
+namespace GTDrawingLink.Components.Text {
+    public class ArrowAttributeComponent : TeklaComponentBase {
+
+
+        public ArrowAttributeComponent()
+            : base(ComponentInfos.ArrowAttributesComponent) {
+        }
+
+        public override GH_Exposure Exposure => GH_Exposure.primary;
+        protected override Bitmap Icon => Resources.MoveView;
```

Comment:
It's better to skip icon than reusing some different.
Default Icon will be clearly visible in the GH ribbon, so it will be easier to address lack of correct icon.